src/pydvl/value/stopping.py

```diff
@@ -623,3 +628,74 @@ def reset(self):
 
     def __str__(self):
         return f"HistoryDeviation(n_steps={self.n_steps}, rtol={self.rtol})"
+
+
+class RankStability(StoppingCriterion):
```

Comment:
I would add a `fraction` parameter ∈ [0,1] to this, to allow looking only at some of the data. Ususally one wants to focus on the top or bottom segment only. For the bottom, maybe allow a negative value for `fraction`?

Reply:
Are you talking about only looking at the highest value fraction of the data or just the first data in the dataset? Sorting the data values, then looking at the highest value fraction does not make sense for checking the order of the values (because if would always be ordered and stop in Iteration 1). How exactly do you envision this feature to work?

Reply:
I meant "check stability among top k% of the data". So, for two runs, take the top k% points and compute ... 🤦🏽 as I'm typing this I realize I was thinking of what we did in the repro: look at the % of points that remain across the top k% positions across runs. Come to think of it, how is Spearman correlation a good stopping criterion? It's always going to be dismal across the whole dataset, isn't it?

Reply:
This criterion would make sense, however, it does not involve the Spearman correlation at all and thus would be an entirely new Stopping criterion. I suggest to open a separate PR for this.